File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/AbreConexaoImpressora.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class AbreConexaoImpressora(
    private val tipo: Int,
    private val modelo: String,
    private val conexao: String,
    private val parametro: Int
) : TermicaCommand("AbreConexaoImpressora") {
    override fun functionParameters(): String {
        return "\"tipo\"" + ":" + tipo + "," +
                "\"modelo\"" + ":" + "\"" + modelo + "\"" + "," +
                "\"conexao\"" + ":" + "\"" + conexao + "\"" + "," +
                "\"parametro\"" + ":" + parametro
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/abre_conexao_impressora.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class AbreConexaoImpressora extends TermicaCommand {
  final int _tipo;
  final String _modelo;
  final String _conexao;
  final int _parametro;

  AbreConexaoImpressora(
      this._tipo, this._modelo, this._conexao, this._parametro)
      : super('AbreConexaoImpressora');

  @override
  functionParameters() {
    return '"tipo"'
        ':'
        '$_tipo'
        ','
        '"modelo"'
        ':'
        '"$_modelo"'
        ','
        '"conexao"'
        ':'
        '"$_conexao"'
        ','
        '"parametro"'
        ':'
        '$_parametro';
  }
}
